### PR TITLE
Fixes #1146: Removes trace level logging. Use debug level logging ins…

### LINKED ISSUE #1146: Remove trace logging level in skupper-router
Move all trace level logging to debug and get rid of the trace logging level.

### PR DESCRIPTION
**Pull request by ganeshmurthy**
…tead of trace. Backward compatible, will set logging level to debug if trace is enabled.

### CODE REVIEW

File: python/skupper_router_internal/management/agent.py [hunk 1]
```diff
@@ -273,8 +273,13 @@ def __str__(self):
 
 
 class LogEntity(EntityAdapter):
-
     def __init__(self, agent, entity_type, attributes=None, validate=True):
+        # This is for backward compatibility, we will remove this after a few releases.
+        if attributes and attributes.get("enable") and "trace" in attributes.get("enable"):
+            enabled = attributes.get("enable")
+            if enabled:
```
Comment (kgiusti) [inline python/skupper_router_internal/management/agent.py:280]: Is this check redundant? attributes.get("enabled") was already checked on line 278

File: python/skupper_router_internal/router/hello.py [hunk 2]
```diff
@@ -18,7 +18,8 @@
 #
 
 from .data import MessageHELLO
-from ..dispatch import LOG_TRACE, LOG_CRITICAL
+from ..dispatch import LOG_DEBUG, LOG_CRITICAL
+from ..dispatch import LOG_DEBUG, LOG_CRITICAL
```
Comment (kgiusti) [inline python/skupper_router_internal/router/hello.py:22]: cut and paste duplicate?

**Review by kgiusti — APPROVED** [review]

File: python/skupper_router_internal/router/hello.py (hunk 2 above)
Comment (ganeshmurthy) [reply python/skupper_router_internal/router/hello.py]: Removed the dup line, thanks

File: python/skupper_router_internal/management/agent.py (hunk 1 above)
Comment (ganeshmurthy) [reply python/skupper_router_internal/management/agent.py]: Yep. Fixed ! Thanks

**Pull request merged**